php/libraries/NDB_Page.class.inc

```diff
@@ -709,6 +709,22 @@ class NDB_Page implements RequestHandlerInterface
             ->withBody(new \LORIS\Http\StringStream($this->display() ?? ""));
     }
 
+    /**
+     * This function can be overridden in a module's page to load the necessary
+     * resources to check the permissions of a user.
+     *
+     * @param User                   $user    The user to load the resources for
+     * @param ServerRequestInterface $request The PSR15 Request being handled
+     *
+     * @return ResponseInterface
+     */
+    public function loadResources(
+        \User $user, ServerRequestInterface $request
+    ) : ResponseInterface {
```

Comment:
There is no reason for this to return anything (and also, the return value isn't used anywhere..)

Reply:
I have it returning something because in the session class, I moved some of the code that was originally found in the `handle` method to `loadResources` and that includes returning exceptions if there is an invalid session or if the session is not found (lines 73-92). 

I can remove this part of the code and try to put it back in the `handle` method if this is preferrable.

Reply:
@driusan In the latest commit, `loadResources` now returns void. I changed the code to throw LorisExceptions (instead of returning a ResponseInterface) if the session id is either non-numeric or does not exist. Let me know if this works better!